### DISCUSSION TITLE
Is pytransitions "production ready"?

### DISCUSSION THREAD

**Discussion by tomtitherington** [pytransitions/transitions#624] [Q&A]
Hi, 

I'd really like to use pytransitions for a commercial project I'm working on and just wondered whether the library is in a stable state and "safe" to use in production since the project has not yet reached version 1.0. 

If I were to use this project, it would be an integral part of the application and as such can't afford to fail... It would be great to hear a little more about the state of the project and whether anyone is using this in production. If so, for how long...etc 

For reference, I'd be using this with the Django (REST) framework. 

Thanks!

**Comment by renewfresho** [comment]
I'm also interested to know what people are doing with pytransitions in terms of scale and fault tolerance. All the examples assume it is running on one machine. Has anyone tried to run it in a cluster. What did they use for distributed locking. etc.
